public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -184,14 +184,26 @@ describe('LokiDatasource', () => {
 
       it('should add volume hint param for log volume queries', () => {
         const target = { expr: '{job="grafana"}', refId: 'B', volumeQuery: true };
-        const req = ds.createRangeQuery(target, options as any, 1000);
-        expect(req.hint).toBe('logvolhist');
+        ds.runRangeQuery(target, options);
+        expect(backendSrv.fetch).toBeCalledWith(
+          expect.objectContaining({
+            headers: {
+              'X-Query-Tag': 'Source=logvolhist',
```

Comment:
Wasn't it supposed to be `X-Query-Tags` I guess it's not a big deal.

Reply:
Fixed in https://github.com/grafana/grafana/pull/42205